### ISSUE TITLE
Incompatible with Google Play services and Firebase 15+ version scheme changes

### ISSUE THREAD

**Issue by jkasten2** [OneSignal/OneSignal-Gradle-Plugin#32]
Google has changed the versioning relationship between `com.google.android.gms:play-services-*` and `com.google.firebase:firebase-*` in version 15+ of these libraries. Firebase versions and Google Play services will now no longer have the same version numbers available between them.
https://android-developers.googleblog.com/2018/05/announcing-new-sdk-versioning.html

This means the following in now an anti pattern:
```gradle
buildscript {
    ext {
        play_version = '15.0.0'
    }
}

dependencies {
    // DON'T DO THIS!!
    // The following use of the above buildscript property is no longer valid.
    implementation "com.google.android.gms:play-services-auth:${play_version}"
    implementation "com.google.firebase:firebase-auth:${play_version}"
    implementation "com.google.firebase:firebase-firestore:${play_version}"
}
```

This plugin will need to be updated to support checking the version dependencies of Firebase independent of Google Play services.
Google's new `com.google.gms:google-services:3.3.0` gradle plugin may also try to do some alignment so this should investigated to if it now applies it own resolution rules. Or does it just cause build failures on mismatches now?

**Comment by JeetSingh04** [comment]
@jkasten2  How did you overcome this?

**Comment by jkasten2** [comment]
@JeetSingh04 This issue was fixed in version [0.10.0](https://github.com/OneSignal/OneSignal-Gradle-Plugin/releases/tag/0.10.0) of this plugin. Make sure you are using this version or newer. Please open a new issue with all the as many details as you can if this is already updated in your project.
